lib/WP_Auth0_Nonce_Handler.php

```diff
@@ -61,9 +61,10 @@ private function __construct() {
 	 * Start-up process to make sure we have something stored.
 	 */
 	protected function init() {
-		if ( defined( static::NONCE_COOKIE_NAME ) && isset( $_COOKIE[ static::NONCE_COOKIE_NAME ] ) ) {
+		// If a NONCE_COOKIE_NAME is not defined then we don't need to persist the nonce value.
+		if ( defined( static::NONCE_COOKIE_NAME ) && isset( $_COOKIE[ self::get_storage_cookie_name() ] ) ) {
 			// Have a cookie, don't want to generate a new one.
-			$this->unique = $_COOKIE[ static::NONCE_COOKIE_NAME ];
+			$this->unique = $_COOKIE[ self::get_storage_cookie_name()  ];
```

Comment:
what about `self::get_cookie_prefix() + static::NONCE_COOKIE_NAME`? Same for the remaining calls

Reply:
No need to exert that kind of control, name is irrelevant.